lstchain/visualization/plot_dl2.py

```diff
@@ -200,7 +200,7 @@ def plot_e(data, n_bins, emin, emax, true_hadroness=False):
     plt.plot(bin_centers,standard_deviations,
              marker='X',linestyle='None')
     plt.ylabel('STD',fontsize=24)
-    plt.xlabel('$log_{10}E_{true}(GeV)$',fontsize=24)
+    plt.xlabel('$log_{10}E_{true}[TeV]$',fontsize=24)
```

Comment:
Should be `r'$\log_{10}(E_\mathrm{true} / TeV)$`

Reply:
done